modules/administration/pages/ssl-certs.adoc

```diff
@@ -26,6 +30,8 @@ Ensure that the intermediate certificate comes first in the combined file.
 By default, {productname} uses a self-signed certificate.
 After you have completed the initial setup, you can replace the default certificate with a custom certificate.
 
+
```

Comment:
Because of back-ports (maybe) let's try avoid white-space changes as much as possible.  Esp. if we are not that sure whether we want to have 3 empty lines before procedure titles or not ;)

Reply:
I agree, trying to add some consistency. I thought we'd agreed on three lines before headings?